examples/hello-world-sandbox/hello-world.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: agents.x-k8s.io/v1alpha1
+kind: Sandbox
+metadata:
+  name: {USERNAME}-hello-world
```

Comment:
It is usually cleaner to just use a static name like `hello-world-sandbox` instead of adding user name to the resource name.